Client/Frontend/Browser/FaviconHandler.swift

```diff
@@ -95,12 +94,14 @@ class FaviconHandler {
 extension FaviconHandler: TabEventHandler {
     func tab(_ tab: Tab, didLoadPageMetadata metadata: PageMetadata) {
         tab.favicons.removeAll(keepingCapacity: false)
-        guard let faviconURL = metadata.faviconURL else {
-            return
-        }
-
-        loadFaviconURL(faviconURL, forTab: tab) >>== { (favicon, data) in
-            TabEvent.post(.didLoadFavicon(favicon, with: data), for: tab)
+        if let iconURL = metadata.largeIconURL {
+            loadFaviconURL(iconURL, type: .appleIcon, forTab: tab) >>== { (favicon, data) in
```

Comment:
What is this custom `>>==` operator?

Reply:
Some Deferred operator I believe, it's just copy pasted from the line below